src/ParameterRegistry.cpp

```diff
@@ -691,6 +694,41 @@ void ParameterRegistry::initialize(Config& config) {
         "Set the logging verbosity level"
     );
 
+    // Debugging Checkboxes
+    if (config.get<int>("system.log_level") == static_cast<int>(Logger::Level::DEBUG)) {
+        addBoolConfigParam(
+            "system.timing_debug.enabled",
+            "Loop timing in console",
+            sSystemSection,
+            1111,
+            &timingDebugActive,
+            "Enable or disable the process loop time debugging in console.<br>"
+            "r=draw display buffer<br>"
+            "D=display refresh<br>"
+            "W=website<br>"
+            "M=MQTT<br>"
+            "H=hassio<br>"
+            "T=temperature"
```

Comment:
You can move the check for the log level into showCondition: 

`, [&config] { return config.get<int>("system.log_level") == static_cast<int>(Logger::Level::DEBUG); }`